xblockutils/base_test.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
```

Comment:
Fair enough. Or disable them per-function, so future functions are prompted for docstrings.